.github/workflows/evm.yml

```diff
@@ -29,43 +29,43 @@ jobs:
 
       - name: Run Forge build
         run: |
-          make test-push0
+          make -C evm push0-test
```

Comment:
minor nit: it's possible to specify the working directory for all the commands here, like so:

https://github.com/wormholelabs-xyz/example-native-token-transfers/blob/8712337bb9ec20e6cf32ab70767789af1d0025f6/.github/workflows/solana.yml#L10-L12